pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -4382,3 +4370,22 @@ func setNodeDisk(
 	}
 	volumeMap[volumeID] = check
 }
+
+// isRegionValid accepts an AWS region name and returns if the region is a
+// valid region known to the AWS SDK
+func isRegionValid(region string) bool {
+	regions := sets.NewString()
```

Comment:
Nit: A set is probably overkill here, vs just checking equality :-)

Reply:
I tend to agree with @justinsb, the intention is clearer when checking equality.

Reply:
I'm not a huge fan of this approach but I don't really have major objections either. I've updated the PR to this end.